static/js/pages/instances/InstanceOverview.tsx

```diff
@@ -103,10 +77,6 @@ const InstanceOverview: FC<Props> = ({ controlTarget, instanceName }) => {
             <th>Description</th>
```

Comment:
While you're at this file, can you remove the Label/Value heading? It seems a bit odd.

Reply:
Done @ [4f62c9e](https://github.com/canonical/lxd-ui/commit/4f62c9e2df86d2b692c4298deac8212f25ffcc02)